frontend/src/lib/components/SaveToDashboard/SaveToDashboardModal.tsx

```diff
@@ -115,14 +79,14 @@ export function SaveToDashboardModal({
 
     return (
         <Modal
-            onOk={save}
+            onOk={(e) => void save(e)}
```

Comment:
Just wondering — what's the reason for this?

Reply:
`save` returns a promise, but `onOk` doesn't want anything to do with it. So to have proper types, I added `void`

Basically you can think of `void` as the reverse-`await`.